### PR TITLE
feat: get node version from .nvmrc file in comment-npm-publish workflow

### PR DESCRIPTION
**Pull request by tradeshift-larsbojensen**

### CODE REVIEW

File: .github/workflows/comment-npm-publish.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
 
       - uses: actions/setup-node@v3
         with:
-          node-version: 14
+          node-version-file: ".nvmrc"
```
Comment (wejendorp) [inline .github/workflows/comment-npm-publish.yml:45]: We might need to add guards, just like the dependency-tree workflow: https://github.com/Tradeshift/actions-workflow-npm/blob/master/.github/workflows/dependency-tree-update.yml#L49-L66
- Ignore the middle step there, but a check for `nvmrc` exists, and a fallback.

Comment (tradeshift-larsbojensen) [reply .github/workflows/comment-npm-publish.yml:53]: I updated

**Review by wejendorp — APPROVED** [review]
Lgtm 👍

File: .github/workflows/comment-npm-publish.yml [hunk 2]
```diff
@@ -41,9 +41,24 @@ jobs:
           git checkout pr-find-commit
 
       - uses: actions/setup-node@v3
+      - name: Check for .nvmrc file root folder
+        continue-on-error: true
```
Comment (JesperTerkelsen) [inline .github/workflows/comment-npm-publish.yml:45]: Continue on error = true ?

Comment (tradeshift-larsbojensen) [reply .github/workflows/comment-npm-publish.yml:45]: Don't ask me, I just copy and paste ;) But I think it means it shouldn't abandon, but carry out the next step which is the fallback to default (14).

Comment (wejendorp) [reply .github/workflows/comment-npm-publish.yml:45]: Yea, because we use the exit code in an if statement, not to fail the build.

**Pull request merged**